src/main/java/org/profit/telegrambot/MaksymenBot.java

```diff
@@ -0,0 +1,169 @@
+package org.example;
+
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.*;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+
+public class MaksymenBot extends TelegramLongPollingBot {
+    @Override
+    public String getBotToken() {
+        return "5533026532:AAHGftCTJqPz0KKdegvqwUWUTUg69eDpfjU";
+    }
+
+    @Override
+    public String getBotUsername() {
+        return "@MaksymZymynTestBot";
```

Comment:
Please, unhardcode the values. (e.g. by moving them out of the code to a resource file)